omod/src/test/java/org/openmrs/module/referenceapplication/page/controller/LoginPageControllerTest.java

```diff
@@ -334,6 +334,33 @@ public void get_shouldRedirectUserToHomeIfAuthenticated() throws Exception {
 		assertNotEquals("redirect:" + redirectUrl, new LoginPageController().get(pageModel, uiUtilsWithoutTimezones, pageRequest, null, null,
 		    appFrameworkService, administrationService));
 	}
+
+	/**
+	 * @see LoginPageController#get(org.openmrs.ui.framework.page.PageModel,
+	 *      org.openmrs.ui.framework.UiUtils, org.openmrs.ui.framework.page.PageRequest, String,
+	 *      org.openmrs.api.LocationService,
+	 *      org.openmrs.module.appframework.service.AppFrameworkService,
+	 *      org.openmrs.api.AdministrationService)
+	 */
+	@Test
+	@Verifies(value = "should redirect user to home when user is authenticated if ? redirectUrl is for openrms, even if only have  '/openrms'", method = "get(PageModel,UiUtils,PageRequest)")
+	public void get_shouldRedirectUserToHomeIfAuthenticatedAndPathForOpenmrsWasGivenOnUrl() throws Exception {
```

Comment:
Can we also improve the test method name?

Reply:
Absolutely, do you have some suggestions? maybe something like:
get_shouldRedirectUserToHomeIfAuthenticatedAndPathUrlAreInOpenmrsScope
get_shouldRedirectUserToHomeIfAuthenticatedAndUrlsForOpenmrs

Reply:
Pick what you see best.

Reply:
@dkayiwa i renamed it to `get_shouldRedirectUserToHomeIfAuthenticatedAndURLPathAreInOpenmrsScope`